packages/rocketchat-ui-message/client/renderMessageBody.js

```diff
@@ -9,7 +9,6 @@ renderMessageBody = function(msg) {
 	}
 
 	const message = RocketChat.callbacks.run('renderMessage', msg);
-
```

Comment:
Do not introduce unnecessary changes.

Reply:
sorry, it was a mistake when i was making tests..